core-bundle/src/Framework/ContaoFramework.php

```diff
@@ -127,6 +118,25 @@ public function setHookListeners(array $hookListeners): void
         $this->hookListeners = $hookListeners;
     }
 
+    public function setRequest(Request $request)
```

Comment:
Why is this necessary?

Reply:
If the framework is bootest before routing (a listener with a higher priority) we cannot initialize some parts of the framework.